core/src/main/java/io/ddf/content/Schema.java

```diff
@@ -398,24 +398,32 @@ public void setData(Object[] data) {
 
 
   /**
-   * The R concept of a column "type", such as STRING, INT, LONG, etc.
+   * The R concept of a column "type", such as STRING, INT, BIGINT, etc.
    */
   public enum ColumnType {
```

Comment:
PS: each engine may use different type system to represent their data, while this enum only assume java type.
Solution: we should make this become Abstract Class. Then for each engine we will subclass with specific implementation of type conversion.